modules/k3s/master/instances_server.tf

```diff
@@ -1,5 +1,5 @@
 resource "aws_db_instance" "db" {
-  count                  = (var.datastore_type == "etcd" || var.external_db == "" || var.external_db == "NULL" ? 0 : (var.external_db != "" && var.external_db != "aurora-mysql" ? 1 : 0))
+  count                  = (var.datastore_type == "etcd" || var.datastore_type == "" || var.external_db == "NULL" ? 0 : (var.external_db != "" && var.external_db != "aurora-mysql" ? 1 : 0))
```

Comment:
`var.datastore_type == ""` is this a valid option?

Reply:
its possible but i am validating that on the code also to error out if so 

so good catch, better remove